woocommerce/class-sv-wc-plugin.php

```diff
@@ -716,7 +716,7 @@ public function add_system_status_php_information( $rows ) {
 					continue;
 				}
 
-				/* translators: As in "Value has been set as [foo], but [bar] is required". Placeholders: %1$s - current value for a PHP setting, %2$s - required value for the PHP setting */
+				/* translators: Context: As in "Value has been set as [foo], but [bar] is required". Placeholders: %1$s - current value for a PHP setting, %2$s - required value for the PHP setting */
 				$note = __( 'Set as %1$s - %2$s is required.', 'woocommerce-plugin-framework' );
```

Comment:
the translator feedback said they didn't understand this line and to provide examples, but it's possible they were looking at an outdated file that didn't have the translators comment